blackbox/docs/sql/administration/privileges.txt

```diff
@@ -0,0 +1,138 @@
+.. highlight:: psql
+.. _administration-privileges:
+
+=======================
+Cluster Wide Privileges
+=======================
+
+.. note::
+
+    Privileges are only part of the CrateDB `Enterprise Edition`_.
+    When the CrateDB Enterprise Edition is disabled, there will be
+    no user privilege checks, and every query will be executed
+    without the validation of permissions.
+
+
+``Permission Types``
+====================
+
+When a user is created, he doesn't have any privileges. The superuser can then
+grant privileges to this user.
+
+Please note that only the superuser can ``GRANT`` and ``REVOKE`` privileges.
+The superuser bypasses all privileges and can access all tables in the cluster.
+Therefor, It is also not possible to ``GRANT`` and ``REVOKE`` privileges to the
+superuser.
+
+User permissions are applied on the whole cluster. There is no option to ``GRANT``
```

Comment:
you use permission and privilege as interchangeable words, but I think we should stick with privileges. @dobe your opinion?